datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/models.py

```diff
@@ -86,8 +86,8 @@ def models(ctx, check, sync, verbose):
     files_failed = {}
     num_files = 0
 
-    license_header_lines = get_license_header().splitlines(True) + ['\n']
-    documentation_header_lines = ['\n'] + get_config_models_documentation().splitlines(True) + ['\n']
+    license_header_lines = get_license_header().splitlines(True) + ['\n', '\n']
```

Comment:
Good catch thanks, double checked and it looks good. 

I also added an extra validation for this custom file (`validators.py`) because an empty line was added at the top of the file in -core

Reply:
I also tested the command in both -core and -extras, generating new files and validating existing ones, looks good.